pyro_risks/train.py

```diff
@@ -126,9 +126,7 @@ def train_pipeline(
 
     elif model == "XGBOOST":
         xgb_pipeline.fit(
-            X_train,
-            y_train,
-            xgboost__eval_metric=cfg.XGB_FIT_PARAMS["eval_metric"],
+            X_train, y_train, xgboost__eval_metric=cfg.XGB_FIT_PARAMS["eval_metric"]
```

Comment:
At some point, we'll have to decide if we go full black style or not :sweat_smile:

Reply:
Not sure to follow,  we have added black check within the CI and they are green